doc/articles/includes/how-tos-and-tutorials-inline-toc.md

```diff
@@ -36,4 +36,4 @@
 
 ## Other examples
 
-* [Uno.Samples](../uno-samples.md)
+* [Uno.Samples](../external/uno.samples/doc/samples.md)
```

Comment:
@morning4coffe-dev It's best to use a xref, if possible.

Reply:
💡 thanks! I didn't know about it, yet.